features/attachments.feature

```diff
@@ -26,7 +26,7 @@ Feature: Attachments
     When I run cucumber-js
     Then scenario "some scenario" "Before" hook has the attachments:
       | DATA     | MEDIA TYPE | MEDIA ENCODING |
-      | iVBORw== | image/png  | BASE64         |
+      | iVBORw== | image/png  | 1              |
```

Comment:
I'd like these left as BASE64 as that is better than 1/0. What prompted this change?

Reply:
This was because of the [message structure changing](https://github.com/cucumber/cucumber/blob/master/messages/messages.proto#L366) - thinking about it, I can probably keep a map in the step code from a string to the TypeScript enum.

Reply:
(this was done)